src/test/kotlin/racingcar/domain/CarTest.kt

```diff
@@ -0,0 +1,33 @@
+package racingcar.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class CarTest {
+
+    @Test
+    fun `should throw exception if the name is over 5 characters`() {
+        assertThrows<IllegalArgumentException> { Car("hyeonj") }
+    }
+
+    @Test
+    fun `should init distance as 0 when only name given`() {
+        val car = Car("hyoen")
+        assertThat(car.distance).isEqualTo(Distance(0))
+    }
```

Comment:
```suggestion
    @Test
    fun `should init distance as 0 when only name given`() {
        val car = Car("hyoen")
        assert(car.distance == Distance(0))
    }
```

Do you prefer `assertThat()` of AssertJ, or `assert()` of Kotlin?